packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -0,0 +1,877 @@
+						await insertNodesToTestTree(provider, tree, i, TreeShape.Wide, dataType);
+					},
+					benchmarkFn: () => {
+						readCursorTree(tree.forest, i, TreeShape.Wide);
+					},
+				});
+			}
+			for (const [i, benchmarkType] of nodesCountDeep) {
+				let tree: ISharedTree;
+				let provider: ITestTreeProvider;
+				benchmark({
+					type: benchmarkType,
+					title: `Deep Tree (${TestPrimitives[dataType]}) with cursor: writes ${i} nodes`,
+					before: async () => {
+						provider = await TestTreeProvider.create(1);
+						tree = provider.trees[0];
+						tree.storedSchema.update(testSchema);
+					},
+					benchmarkFn: async () => {
+						await insertNodesToTestTree(provider, tree, i, TreeShape.Deep, dataType);
+					},
+				});
+			}
+			for (const [i, benchmarkType] of nodesCountWide) {
+				let tree: ISharedTree;
+				let provider: ITestTreeProvider;
+				benchmark({
+					type: benchmarkType,
+					title: `Wide Tree (${TestPrimitives[dataType]}) with cursor: writes ${i} nodes`,
+					before: async () => {
+						provider = await TestTreeProvider.create(1);
+						tree = provider.trees[0];
+						tree.storedSchema.update(testSchema);
+					},
+					benchmarkFn: async () => {
+						await insertNodesToTestTree(provider, tree, i, TreeShape.Wide, dataType);
+					},
+				});
+			}
+			describe(`Edit Cursor Tree ${TestPrimitives[dataType]}`, () => {
+				for (const [i, benchmarkType] of nodesCountDeep) {
+					let tree: ISharedTree;
+					let provider: ITestTreeProvider;
+					let path: UpPath;
+					benchmark({
+						type: benchmarkType,
+						title: `Update value at leaf of ${i} deep tree`,
+						before: async () => {
+							provider = await TestTreeProvider.create(1);
+							tree = provider.trees[0];
+							tree.storedSchema.update(testSchema);
+							await insertNodesToTestTree(
+								provider,
+								tree,
+								i,
+								TreeShape.Deep,
+								dataType,
+							);
+							path = getCursorLeafNode(i, TreeShape.Deep);
+						},
+						benchmarkFn: () => {
+							manipulateCursorTree(tree, path, dataType);
+						},
+					});
+				}
+				for (const [i, benchmarkType] of nodesCountWide) {
+					let tree: ISharedTree;
+					let provider: ITestTreeProvider;
+					let path: UpPath;
+					benchmark({
+						type: benchmarkType,
+						title: `Update value at leaf of ${i} wide tree`,
+						before: async () => {
+							provider = await TestTreeProvider.create(1);
+							tree = provider.trees[0];
+							tree.storedSchema.update(testSchema);
+							await insertNodesToTestTree(
+								provider,
+								tree,
+								i,
+								TreeShape.Wide,
+								dataType,
+							);
+							path = getCursorLeafNode(i, TreeShape.Wide);
+						},
+						benchmarkFn: () => {
+							manipulateCursorTree(tree, path, dataType);
+						},
+					});
+				}
+			});
+		}
+	});
+	describe("EditableTree", () => {
+		for (let dataType = 0 as TestPrimitives; dataType <= 4; dataType++) {
+			for (const [i, benchmarkType] of nodesCountDeep) {
```

Comment:
Its not super easy to tell what "i" means here. Picking a more descriptive name, like `depth` or `numberOfNodes` would improve readability. Generally single letter variable names should be avoided, and names made of english words should be used instead.

Reply:
If `i` were just a loop variable used as a counter as its traditionally used, it might be ok here, but even that case more descriptive names are preferred.

Reply:
renamed `i` to `numberOfNodes`